src/screens/NestedInfoScreen.tsx

```diff
@@ -0,0 +1,118 @@
+import { StackScreenProps } from '@react-navigation/stack';
+import React, { useCallback, useContext, useState } from 'react';
+import { ActivityIndicator, RefreshControl, SectionList } from 'react-native';
+
+import {
+  HtmlView,
+  LoadingContainer,
+  LoadingSpinner,
+  SafeAreaViewFlex,
+  SectionHeader,
+  Wrapper,
+  WrapperWithOrientation
+} from '../components';
+import { colors } from '../config';
+import { useStaticContent } from '../hooks';
+import { useRenderItem } from '../hooks/listHooks';
+import { NetworkContext } from '../NetworkProvider';
+import { QUERY_TYPES } from '../queries';
+
+type NestedInfo = {
+  content?: string;
+  title: string;
+  children: Array<{
+    title: string;
+    routeName: string;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    params?: any;
+  }>;
+};
+
+const ListHeaderComponent = ({ loading, html }: { loading: boolean; html?: string }) => {
+  if (loading) {
+    return <LoadingSpinner loading />;
+  }
+
+  if (!html?.length) {
+    return null;
+  }
+
+  return (
+    <Wrapper>
+      {/* @ts-expect-error HtmlView uses memo in js, which is not inferred correctly */}
+      <HtmlView html={html} />
+    </Wrapper>
+  );
+};
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const NestedInfoScreen = ({ navigation, route }: StackScreenProps<any>) => {
```

Comment:
was this screen based on another one? are lines copied from another screen?

Reply:
i don't remember. is there anything wrong with this line?

Reply:
no, i thought of how i review, if i compare to another screen or not

Reply:
it probably still is helpful to do that